tests/go.sum

```diff
@@ -131,8 +131,6 @@ github.com/pkg/errors v0.9.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINE
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4/go.mod h1:xMI15A0UPsDsEKsMN9yxemIoYk6Tm2C1GtYGdfGttqA=
-github.com/rancher-sandbox/ele-testhelpers v0.0.0-20251113092326-9b43cc34d982 h1:pQ0S4n8+impomKiOYBXuOGKGC6/8TV0rbFNLOhGtUFY=
```

Comment:
sorry. what is the consecuence of this removal?
Shouldn't be done in other pr?

Reply:
No side effect of it, as it is just removal.

If it was addition then separate PR is required.